parts/k8s/kubernetesconfigs.sh

```diff
@@ -198,10 +211,15 @@ configureCNI() {
         fi
         /sbin/ebtables -t nat --list
     fi
-    if [[ "${NETWORK_PLUGIN}" = "cilium" ]]; then
-        systemctl enable sys-fs-bpf.mount
-        systemctl restart sys-fs-bpf.mount
-        REBOOTREQUIRED=true
+}
+
+configureCNIIPTablesAKS() {
+    if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
+        if [[ "${NETWORK_POLICY}" != "calico" ]]; then
+            mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
```

Comment:
Maybe quote and squiggle-bracket `$CNI_BIN_DIR`? (We should bite the bullet and `shellcheck` all these at some point.)